cumulusci.yml

```diff
@@ -256,7 +256,17 @@ flows:
                 task: test_data_relationships
             9:
                 task: deploy_reports
-    
+
+    qa_org_namespaced:
+        description: Execute the qa_org flow against a namespaced scratch org and patch help text.
+        steps:
+            1:
+                flow: qa_org
+            2:
+                task: deploy
+                options:
+                    path: unpackaged/config/qa_namespaced
```

Comment:
You shouldn't need a separate metadata folder for this. Put `%%%NAMESPACED_ORG%%%` in unpackaged/config/qa in the places where we need the npsp__ prefix for namespaced orgs but no prefix for unnamespaced orgs. Then use the `deploy_qa_config` task with the `namespaced_org: True` option set.

Reply:
@davidmreed oh, just saw your comment. What other tokens were getting injected?

Reply:
@davisagli The existing `qa` metadata bundle has other tokens like, for example, `___NAMESPACE___Data_Import__c`. If those are injected with `npsp__`, they won't deploy to the namespaced org.  `zip_inject_namespace()` maps the `%%%NAMESPACED_ORG%%%` token to blank if `managed` is not `True`, but if `managed` is `True`, all other tokens also get mapped to the namespace.

Does that make sense, or am I missing something critical?

Reply:
@davidmreed that may be a bug in the namespace injection code. I should take a closer look; in the meantime I'm okay with this workaround.

Reply:
Quick update, I am still getting the error while trying to create a bge batch. Will troubleshoot and update